landingzones/views.py

```diff
@@ -221,6 +222,70 @@ def submit_create(
             raise ex
 
 
+class ZoneUpdateMixin(ZoneConfigPluginMixin):
+    """Mixin to be used in zone update in UI and REST API views"""
+
+    def submit_update(self, zone, request=None):
```

Comment:
This can be renamed to e.g. `update_zone()`, as we are not doing any async submission here.